handbook/ce/index.md

```diff
@@ -12,7 +12,8 @@ We work with many other teams at Sourcegraph, including:
 
 ---
 
-* [Onboarding](onboarding.md)
+* [Customer Engineering Onboarding](onboarding.md)
+* [Customer Support Onboarding] (customer-support-onboarding.md)
```

Comment:
@virginiaulrich Need to remove the space between `] (` on line 16! :)

Reply:
@emchap updated :)